source/apis/search/fields.html.md.erb

```diff
@@ -2,12 +2,8 @@
 layout: api_layout
 title: Search API field reference
 parent: /search-api.html
-source_url: https://github.com/alphagov/rummager/blob/master/config/schema/field_definitions.json
+source_url: https://github.com/alphagov/search-api/blob/master/config/schema/field_definitions.json
 ---
 
-Documents in an Elasticsearch index have a type, and each type can have different fields.
```

Comment:
We do still have different types which have different fields, with search-api using the schemas for validation, but they map down to the same elasticsearch type.  That's fairly confusing and I think should be brought up here.